earthquakes/engineering.py

```diff
@@ -8,7 +8,6 @@
 
 def sequence_generator(data, xcol="acoustic_data", ycol="time_to_failure", size=150000):
     """Generator that extracts segments of the signal from the data.
-
```

Comment:
You seem to have removed all the white space in the docstrings, which makes them a bit hard to read. Can you re-add the empty lines before `Parameters`, `Returns`, etc.?